cmd/vic-machine/common/utils.go

```diff
@@ -57,8 +59,30 @@ func CheckUnsupportedCharsDatastore(s string) error {
 func checkUnsupportedChars(s string, re *regexp.Regexp) error {
 	st := []byte(s)
 	var v []int
+	//this is validation step for characters in a datastore URI
```

Comment:
haha thanks :) I have been having randomissues with my `gofmt` not doing this on save for me.